src/runtime/messages.ts

```diff
@@ -14,6 +15,18 @@ export type LocaleLoader<T = LocaleMessages<DefineLocaleMessage>> = {
   load: () => Promise<MessageLoaderResult<T>>
   cache: boolean
 }
+
+/**
+ * Used for runtime debug logs only, cannot be imported due to
+ * this being a virtual file in both nuxt and nitro contexts
+ *
+ * NOTE: We could export the logger from a virtual file instead
+ */
+const debugLogger = /*#__PURE__*/ createConsola({ level: __DEBUG_VERBOSE__ ? 999 : 4 }).withTag('i18n')
+function createLogger(label: string) {
```

Comment:
```suggestion
/* @__NO_SIDE_EFFECTS__ */
function createLogger(label: string) {
```

Reply:
That doesn't seem to work either 🤔 

I'm using `/*#__PURE__*/` wherever the `createLogger` function is called to achieve the same though, from what I understand `/* @__NO_SIDE_EFFECTS__ */` would not work for projects using webpack right?